charts/fluentd-elasticsearch/templates/pod-security-policy.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.podSecurityPolicy.enabled }}
-apiVersion: extensions/v1beta1
+apiVersion: policy/v1beta1
```

Comment:
please use somthing like: https://github.com/kiwigrid/helm-charts/blob/master/charts/fluentd-elasticsearch/templates/daemonset.yaml#L7

Otherwise earlier K8s versions might not work anymore.

Reply:
i understand the reason. unfortunately this wouldn't fix my problem because argo do not honor this with 1.3 https://github.com/argoproj/argo-cd/issues/1251

maybe a minor bump of chart version is good enough, to mark it as incompatible?

Reply:
Bump chart Version and restrict kubeVersion? 
https://v2.helm.sh/docs/charts/#charts

kubeVersion: A SemVer range of compatible Kubernetes versions (optional)

Reply:
we wont change our charts to make third party ci work.

breaking changes of the chart would need a major version bump but im not willing to break somehting here as its not needed when api version is set according to kubernetes version.

Reply:
@monotek This api is available since v1.10 (I hope there is nobody out there running an older k8s).

Reply:
ok